src/pages/requests/index.tsx

```diff
@@ -15,17 +15,64 @@ import { layerStyle } from "../../others/components/map/CircleLayerStyle";
 import { mapAidRequestsToFeatures } from "../../others/helpers/map-utils";
 import { processAidRequests } from "../../others/helpers/process-aid-request";
 import { useSidebarContext } from "../../others/components/sidebar-context";
+import { FilterItem, useFilter } from "../../others/contexts/filter";
 
 export function Requests() {
   const { t } = useTranslation();
   const { data: cities } = useLocationsQuery();
   const { data: supplies } = useSuppliesQuery();
   const { data: aidRequests } = useAidRequestQuery();
+  const filterContext = useFilter();
```

Comment:
nit:

```suggestion
  const { addFilter } = useFilter();
```

then L27 can be removed